### PR TITLE
368: Computed Date Refactoring

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: shared/src/persistence/dynamo/workitems/saveWorkItemForPaper.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const { put } = require('../../dynamodbClientService');
  * @returns {Promise} the promise for the call to persistence
  */
 exports.saveWorkItemForPaper = async ({ applicationContext, workItem }) => {
+  console.log('le workItem :D', workItem);
```
Comment (matthopson) [inline shared/src/persistence/dynamo/workitems/saveWorkItemForPaper.js]: Le remove?

Comment (sutt0n) [reply shared/src/persistence/dynamo/workitems/saveWorkItemForPaper.js]: Le done 😄

**Review by matthopson — APPROVED** [review]
Good work, man!

**Comment by sutt0n** [comment]
> Good work, man!

Thank you, and to you as well, man! 😄

**Pull request merged**